frontend/src/metabase/databases/components/DatabaseConnectionUri/engines-config.ts

```diff
@@ -0,0 +1,116 @@
+import type { EngineKey } from "metabase-types/api/settings";
+
+export const enginesConfig = new Map<EngineKey, { placeholder: string }>([
```

Comment:
There's nothing wrong with this, but why not just define an object? wouldn't it be more concise? 

```ts
{
  athena: { placeholder: "jdbc:athena://WorkGroup=primary;Region=us-east-1;" },
  "biquery-cloud-sdk": { placeholder: "jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=MyBigQueryProject;OAuthType=1;", 
  // etc..
}
```